module-cheatsheets/README.md

```diff
@@ -0,0 +1,8 @@
+### Cheatsheets Module
```

Comment:
I think this content makes more sense as part of CONTRIBUTING.md than living here.